nb/welcome/src/org/netbeans/modules/welcome/resources/tutorial4.url

```diff
@@ -1 +1 @@
-http://netbeans.org/kb/trails/php.html?utm_source=netbeans&utm_campaign=welcomepage
\ No newline at end of file
+https://netbeans.apache.org/kb/docs/php/index.html
```

Comment:
Hi @geertjanw , @neilcsmith-net , what about keeping the the "?utm_source=netbeans&utm_campaign=welcomepage" suffix even in the new URL? Who knows, maybe we can detect requests in the future...